src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSourceUnitTest.java

```diff
@@ -89,13 +118,23 @@ public void testHeadersAreStripped() {
         }
     }
 
+    @Test(groups = "spark", expectedExceptions=UserException.class)
+    public void testReject2BitCRAMReference() {
+        doLoadReadsTest(NA12878_chr17_1k_CRAM, b37_2bit_reference_20_21, ValidationStringency.STRICT);
+    }
+
+    @Test(groups = "spark", expectedExceptions=UserException.class)
+    public void testCRAMReferenceRequired() {
+        doLoadReadsTest(NA12878_chr17_1k_CRAM, null, ValidationStringency.STRICT);
+    }
+
     @Test
     public void testPartitionSizing(){
 
         String bam = dirBQSR + "HiSeq.1mb.1RG.2k_lines.alternate.bam"; //file is ~220 kB
         JavaSparkContext ctx = SparkContextFactory.getTestSparkContext();
 
-        ReadsSparkSource readSource = new ReadsSparkSource(ctx);
+        ReadsSparkSource readSource = new ReadsSparkSource(ctx, ValidationStringency.STRICT);
```

Comment:
I think that all of these tests (except the ones testing the effect of validation stringency itself) should use the one-arg `ReadsSparkSource` constructor (to use the default validation stringency) instead of passing in `STRICT` or `SILENT` explicitly. It's better to test that we can pass these tests with the default stringency (whatever it may be) rather than burdening test authors with having to tailor the stringency to each input.